### PR TITLE
sentry changes

### PR DESCRIPTION
**Pull request by emre2038**
## Description
* sentry on frontend should not be firing events in local dev
* sentry on frontend should distinguish prod events from deploy preview events
* google analytics should not be logging outside of prod (not in deploy previews or locally)

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-222

## How is it tested automatically?
this is not tested automatically

## How should a reviewer test it locally
* click the link to the deploy preview
* open the developer console
* confirm that there is a warning message `Not initialising Google Analytics because GOOGLE_ANALYTICS_ID is not set` because we do not log the events in deploy preview
* confirm that there is NO warning message `Not initialising Sentry because SENTRY_DSN is not set` 
* navigate to a geography that does not exist by changing the url so that the site breaks. for example https://deploy-preview-580--wazimap-staging.netlify.app/?dev-tools=true#geo:xyz
* go to sentry.io and confirm that this error is logged to sentry and the environment is `FE_Deploy-preview`

### Added
* a `toml` file to configure sentry

### Updated
* `index.js` to read configurations from the `toml` file

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

**Review by michaelglenister — CHANGES_REQUESTED** [review]
SENTRY is not initialized in local but it looks like Google Analytics is still set in the deploy preview

image

File: src/js/index.js
Comment (michaelglenister) [inline src/js/index.js:121]: "undefined" in quotes could be what's caused analytics to show up where it shouldn't

Comment (emre2038) [reply src/js/index.js:121]: @michaelglenister "undefined" in quotes was not the problem but looks like in index.html there is some google analytics related script hardcoded

image

Comment (emre2038) [reply src/js/index.js:121]: but since we don't init Analytics class, the events will not be caught and they will not be logged to GA

**Comment by emre2038** [comment]
removed the hardcoded google analytics related scripts in index.html and move them into index.js. we append those scripts only if `GOOGLE_ANALYTICS_ID` key is set.

BEWARE : Webflow export is not updated and the changes in index.html are made manually. This PR must not be merged until Webflow is updated and the export is merged into this PR

**Review by michaelglenister — COMMENTED** [review]
Looks good, ready to merge once the Webflow export is done

**Comment by emre2038** [comment]
webflow export is imported @michaelglenister should I move this into QA?

**Pull request merged**
